docs/get-started/Velox.md

```diff
@@ -264,6 +264,44 @@ spark.celeborn.storage.hdfs.dir hdfs://<namenode>/celeborn
 spark.dynamicAllocation.enabled false
 ```
 
+## Uniffle support
+
+Uniffle with velox backend supports [Uniffle](https://github.com/apache/incubator-uniffle) as remote shuffle service. Currently, the supported Uniffle versions are `0.8.0` and `0.9.0`.
```

Comment:
The supported for Uniffle version `0.9.0` is still working in #6484 , It is better to remove the `0.9.0` if it cannot be merged in `1.2.0`.

Reply:
ok